### PR TITLE
Added DMOJ Rich Presence

### PR DESCRIPTION
**Pull request by Yolodude25**
I added a rich presence for the online coding judge site, DMOJ.
Imgur album of screenshots:

https://imgur.com/a/KZA6kgP

### CODE REVIEW

File: websites/D/DMOJ/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "author": {
+        "name": "yolo!",
+        "id": "277507281652940800"
+    },
+    "url": "dmoj.ca",
+    "description": {
+        "en": "The DMOJ is a modern contest platform and archive of programming problems."
+    },
+    "service": "DMOJ",
+    "version": "2.1.0",
```
Comment (Bas950) [inline websites/D/DMOJ/dist/metadata.json:12]: version 1.0.0 on release

File: websites/D/DMOJ/presence.ts [hunk 2]
```diff
@@ -0,0 +1,191 @@
+var presence = new Presence({
+    clientId: "754070047193956492", //The client ID of the Application created at https://discordapp.com/developers/applications
+});
+
+var browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/D/DMOJ/presence.ts]: No var, remove comment

**Review by Bas950 — CHANGES_REQUESTED** [review]
Dont upload js file

**Comment by Yolodude25** [comment]
> Dont upload js file

deleted the presence.js, and updated presence.ts to follow the codacy checks.

File: websites/D/DMOJ/presence.ts [hunk 3]
```diff
@@ -0,0 +1,191 @@
+const presence = new Presence({
+    clientId: "754070047193956492",
+});
+
+const browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/D/DMOJ/presence.ts:5]: merge consts

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/D/DMOJ/presence.ts (hunk 3 above)
Comment (Yolodude25) [reply websites/D/DMOJ/presence.ts:5]: How would i go about doing this? (I'm new to typescript/javascript so i don't know much about the technicalities)

Would it be something like

```ts
const presence = new Presence({
    clientId: "754070047193956492",
    browsingStamp: Math.floor(Date.now() / 1000);
});
```

Comment (Bas950) [reply websites/D/DMOJ/presence.ts:5]:
```
const presence = new Presence({
    clientId: "754070047193956492",
}),
 browsingStamp = Math.floor(Date.now() / 1000);
```

**Comment by Yolodude25** [comment]
Hi, it seems that the pr quality review is flagging it as not up to standards because I have windows line endings, how should I fix this?

**Review by Bas950 — APPROVED** [review]

**Review by Alanexei — APPROVED** [review]

**Pull request merged**